config.json

```diff
@@ -1,7 +1,7 @@
 {
-  "owner":"",
-  "repo":"",
+  "owner":"swx457056",
+  "repo":"https://github.com/swx457056/test-ci-bot",
   "git_hub_token":"",
-  "webhook_secret":"",
-  "circle_ci_token":""
-}
\ No newline at end of file
+  "webhook_secret":"abcd"
```

Comment:
Is this webhook_secret valid? And I don't think we can put the any secret in source codes!